docs/bootstrap.md

```
@@ -38,13 +38,13 @@ kubectl get secret -n gitea gitea-admin-credentials \
   -o jsonpath='{.data.password}' | base64 --decode
 ```
 
-### Layer 2.5 — Gateway API CRDs + Crossplane Core
-- Installs **Gateway API CRDs** (required before Traefik's GatewayClass can register)
+### Layer 2.5 — Crossplane Core
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)